samples/highcharts/demo/polar-advanced/demo.js

```diff
@@ -0,0 +1,351 @@
+                    color: '#444444'
+                },
+                text: 'Teams'
+            }
+        },
+
+        series: [
+            ...teamSeries,
+            {
+                name: 'Month',
+                type: 'column',
+                data: weekLabels,
+                colorKey: 'x',
+                xAxis: 2,
+                yAxis: 2,
+                showInLegend: false,
+                pointPlacement: 'between',
+                pointPadding: 0,
+                groupping: false,
+                centerInCategory: true,
+                enableMouseTracking: false
+            }, {
+                name: 'Total',
+                type: 'columnrange',
+                data: scoreData,
+                xAxis: 1,
+                yAxis: 1,
+
+                shadow: true,
+                showInLegend: false,
+                centerInCategory: true,
+                pointPlacement: 'on',
+                colorAxis: 2,
+                groupPadding: 0,
+                pointPadding: 0,
+                colorKey: 'x',
+                pointStart: 1,
+                borderColor: 'blueviolet',
+                borderWidth: 0.35,
+                tooltip: {
+                    headerFormat: (
+                        '<span style=\"display:flex; justify-content: center; align-items:center; flex-direction:column;\">' +
+                        '<b style=\"padding:0.4rem; font-size: 2.6rem; letter-spacing: 0.208rem; text-decoration: underline solid {point.color} 0.2rem; text-align:center;\"> Day {point.x}</b>'
+                    ),
+
+                    pointFormat: (
+                        '<span style=\"font-size: 0.9rem;padding:0.3rem;width: 5rem; text-align:left\"><span style=\"color:{point.color}; font-size: 1rem;\">●</span> ' +
+                        '<b>Sales: </b><span>{point.high}</span></span>' +
+                        '<span style=\"font-size: 0.9rem;padding:0.3rem;width: 5rem;text-align:left\"><span style=\"color:{point.color}; font-size: 1rem;\">●</span><b> Average: </b><span>{point.avg}</span></span>' +
+                        '<span style=\"font-size: 0.9rem;padding:0.3rem;width: 5rem; text-align:left\"><span style=\"color:{point.color}; font-size: 1rem;\">●</span> ' +
+                        '<b>Highscore: </b><span>{point.highscore}</span></span>' +
+                        '<span style=\"font-size: 0.9rem;padding:0.3rem;width: 5rem; text-align:left\"><span style=\"color:{point.color}; font-size: 1rem;\">●</span> ' +
+                        '<b>Top earner: </b><span>{point.topEarner}</span></span>'
+                    ),
+
+                    footerFormat: '<i style=\"margin:0.3rem; padding:0.3rem;width: 10rem;text-align:center; border-top: 0.1rem solid {point.color};\">Week {point.week}</i></span>'
+                }
+            }
+        ]
+    }),
+    resizeObserver = new ResizeObserver(() => {
+        chart.legend.update({ y: chart.chartHeight / 12 });
```

Comment:
legend.y as string with % - e.g. `y: '8%'` could remove the need for the observer, right? That could be a feature request - add support for percentage based values for the legend y position.

Reply:
Can't we just set a static value in pixels in `legend.y` property?

Reply:
Thanks for good feedback, Hubert!

The reason I have not done so is that a pixel-value will not work for all sizes, as the chart shrinks/grows the legend becomes missplaced. The legends Y-property does not have a percentage (yet but i will make a feature request as Kacper suggests) thus I chose to do it with the ResizeObserver

Originally I wanted to do it with a callback using HCs responsive system but since I want this rule on every size, I had to put in no conditions, which caused recursion error for some reason :S

Edit:
Putting a minimum condition that all charts satisfy (such as minWidth: 1) would cause the chart not to update the legends position on resize so that too was not an option.

Reply:
Temp fix until ive made a feature req for legend.y as percentage